content/get-started/using-github/keyboard-shortcuts.md

```diff
@@ -94,7 +94,7 @@ If you view a code file in a repository using the **Code** view and click on any
 | Keyboard shortcut | Description
 |-----------|------------
 |<kbd>Shift</kbd>+<kbd>J</kbd>| Highlights the line that is currently selected by the cursor within the code file
-|<kbd>Shift</kbd>+<kbd>F10</kbd> | If a line of code is currently selected, this shortcut opens the line menu for that line, appearing under {% octicon "kebab-horizontal" aria-label="The horizontal kebab icon" %} to the left of the line
+|<kbd>Option</kbd> (Mac)/<kbd>Alt</kbd> (Windows) + <kbd>Shift</kbd>+<kbd>C</kbd> | If a line of code is currently selected, this shortcut opens the line menu for that line, appearing under {% octicon "kebab-horizontal" aria-label="The horizontal kebab icon" %} to the left of the line
```

Comment:
This isn't expressed in the way we normally do things. Better to keep the whole key combo together:

```suggestion
|<kbd>Shift</kbd>+<kbd>Option</kbd>+<kbd>C</kbd> (Mac) or </br> <kbd>Shift</kbd>+<kbd>Alt</kbd>+<kbd>C</kbd> (Windows/Linux) | If a line of code is currently selected, this shortcut opens the line menu for that line, appearing under {% octicon "kebab-horizontal" aria-label="The horizontal kebab icon" %} to the left of the line
```